lib/components/app/app-menu.js

```diff
@@ -53,6 +56,11 @@ class AppMenu extends Component {
           <MenuItem onClick={this._showRouteViewer}>
             <Icon type='bus' /> {languageConfig.routeViewer || 'Route Viewer'}
           </MenuItem>
+          {mailablesEnabled &&
+            <MenuItem onClick={this._toggleMailables}>
+              <Icon type='envelope' /> Mailables
+            </MenuItem>
+          }
```

Comment:
It seems somewhat inconsistent to have the mailables be activated here while there are these special buttons to activate the field trip and call windows. Perhaps the call taker and field trip windows should also be toggleable in here too?

Reply:
Yea, I see what you're saying. The mailables menu item here was requested by TriMet, but I do like the idea of also showing the Call/Field Trip items here as well.

Reply:
Added in https://github.com/opentripplanner/otp-react-redux/pull/378/commits/8ae80cb6da5e63712a54eedcd21a50ab18c5efd5